### PR TITLE
Hide "Show health pills" when debugger is inactive

### LINKED ISSUE #318: The "Show health pills" toggle in the graph dashboard should be hidden.
That toggle is currently a no-op as far as the world is concerned.
kqedgeetglh
Thank you to @wchargin for bringing this up.

### PR DESCRIPTION
**Pull request by chihuahua**
We now hide the "Show health pills" toggle within the graph dashboard
when the debugger plugin is inactive.

When the debugger is active:
download

vs inactive:
qxdnr7judre

Previously, the problem had been that a dom-if block had been used as a
direct child of a `<table>` element. Apparently, polymer ignores the
dom-if condition in that case. I also formatted the toggles using divs
instead of tables and positioned labels to the right of their respective
toggle widgets for alignment.

Fixes #318.

### CODE REVIEW

File: tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html [hunk 1]
```diff
@@ -619,7 +613,7 @@
               </defs>
               <path marker-end="url(#dataflow-arrowhead-legend)"
                     stroke="#bbb" d="M2 9 l 29 0"
-                    stroke-linecap="round" />
+                    stroke-linecap="round" />showUploadButton
```
Comment (wchargin) [inline tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html]: Remove?

**Comment by chihuahua** [comment]
Yes! The toggle appears when the debugger plugin is active.

4h1rcvh4nnm

**Comment by wchargin** [comment]
Reapproved, then!

File: tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html (hunk 1 above)
Comment (chihuahua) [inline tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html]: Done.

**Pull request merged**